core/src/main/scala/org/apache/spark/deploy/SparkSubmitArguments.scala

```diff
@@ -56,8 +55,16 @@ private[spark] class SparkSubmitArguments(args: Seq[String], env: Map[String, St
   var verbose: Boolean = false
   var isPython: Boolean = false
   var pyFiles: String = null
+  var action: SparkSubmitAction = null
   val sparkProperties: HashMap[String, String] = new HashMap[String, String]()
 
+  // Standalone cluster mode only
+  var useRest: Boolean = true
```

Comment:
this is not actually settable by the user, so for this one it might be good to indicate it's used only internally